libs/state/schematics/utils/npm.ts

```diff
@@ -0,0 +1,36 @@
+import { get } from 'http';
+
+export interface NpmRegistryPackage {
+  name: string;
+  version: string;
+}
+
+export function getLatestNodeVersion(
+  packageName: string
+): Promise<NpmRegistryPackage> {
+  const DEFAULT_VERSION = 'latest';
+
+  return new Promise((resolve) => {
```

Comment:
I'd suggest keeping track of the versions in a local file instead of checking the npm registry.

Reply:
The good part with checking the npm registry is that we don't need to manually update the package version on each release.